styles/palette.styl

```diff
@@ -18,7 +18,6 @@ $footerColor = #828282
 $newsletterBgColor = #f8f8f8
 
 // layout
-$navbarHeight = 5rem
```

Comment:
If we are deleting vars maybe we should delete them all and let a css customisation override But delete some and let some what do you think about it ?

Reply:
Because the variable is not semantic, it's just copy from vuepress' default palette. IMO, it should be `headerHeight`. 

Nevertheless, in this #61, I've added a new variable `headerHeight`.

Reply:
Ok

Reply:
Actually, before I did the change, header is not using any variable as its height. Its height is 80px since this theme is created (designed).  

Next, 5rem is a magic number and has never been set to header, so I add  `$headerHeight ` with the correct value 80px, and make header use it so that people can override it.

Reply:
ok i get it but if we want to stay consistent rem is a better practice no ?

Reply:
okay then I'll check out source code to see is there any other styles written with px should be refactored to use rem instead.  Thanks the review.